### ISSUE TITLE
Setup and configure Jenkins with automatic updates

### ISSUE THREAD

**Issue by adborden** [GSA/datagov-ckan-multi#120]
## User Story
As an operator, I want Jenkins to be configured with automatic updates so that when an update is available for Jenkins or one of its plugins, the Jenkins instance is rebuilt or updated automatically without human intervention.

## Details / tasks
Looks like there are some security updates available for Jenkins that are not being applied.
![Screenshot from 2019-08-15 06-54-10](https://user-images.githubusercontent.com/509703/63099673-6412dc00-bf2a-11e9-9c16-c40a0c6d06fd.png)

## Acceptance Criteria
- [ ] When updates are available to Jenkins or plugins, the system will pick them up automatically

**Comment by pwalsh** [comment]
@adborden would you feel that this story is sufficiently covered by https://github.com/datopian/ckan-cloud-operator/issues/82 ? 

In general, I am not sure we want automatic updates of dependencies, but rather, that when CCO dependencies get updates, a team verifies them, and build a new image of the "management server" which can then be deployed (so, dependencies updates are first verified to pass some integration tests etc.).

Let me know what you think.

**Comment by adborden** [comment]
Yes, but what event would notify the developer that it's time to rebuild the image with updated dependencies (specifically, Jenkins or its plugins)?
